### PR TITLE
Added: examples to features page

### PR DESCRIPTION
**Pull request by magsout**
the descriptions and examples were taken from the `README.md` of each plugin

### CODE REVIEW

**Comment by MoOx** [comment]
Thanks for the initiative.
I think we should:
- not use "PostCSS plugin" in every description
- add browser scope when relevant
- remove link from titles because incoming toc will make some shit will all those links
- add newlines below title & before example
- maybe update examples a bit, to reflect more real world usage ?
- maybe generate output (in a tab ?) using the playground script ?

(Note: when you submit a PR, use "review needed" label since "needs revision" is when some reviewers ask for some updates ^^)

**Comment by magsout** [comment]
Ok thanks for the review

File: docs/content/features.md [hunk 1]
```diff
@@ -4,94 +4,389 @@ backgroundModifier: darkRoad
...
+  font-variant-numeric: lining-nums;
+}
+
+```
+
+## [filter](http://www.w3.org/TR/filter-effects/)([⇗](https://github.com/iamvdo/pleeease-filters))
+[PostCSS](https://github.com/postcss/postcss) plugin to transform CSS shorthand filters to SVG equivalent.
+```css
+.blur {
+	filter: blur(4px);
```
Comment (bloodyowl) [inline docs/content/features.md]: tabs

File: docs/content/features.md [hunk 2]
```diff
@@ -4,94 +4,389 @@ backgroundModifier: darkRoad
...
+}
+```
+you will get
+
+```css
+.blur {
+	filter: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="filter"><feGaussianBlur stdDeviation="4" /></filter></svg>#filter');
```
Comment (bloodyowl) [inline docs/content/features.md]: tabs

File: docs/content/features.md (hunk 1 above)
Comment (magsout) [inline docs/content/features.md]: ah yes, thanks, fucking copy paste

File: docs/content/features.md (hunk 2 above)
Comment (magsout) [inline docs/content/features.md]: ah yes, thanks, fucking copy paste

**Comment by MoOx** [comment]
Well... I forgot this PR when I released 1.7.0...
What do you think of the current page ? http://cssnext.io/features/

**Comment by magsout** [comment]
wooot fucking nice. Awesome, good job.

**Pull request closed without merging**